packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1159,7 +1159,13 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
               textCapitalization: widget.textCapitalization,
               keyboardAppearance: widget.keyboardAppearance,
           ),
-      )..setEditingState(localValue);
+      )
+      ..setEditingState(localValue)
+      ..addListener(() {
+        if (!_textInputConnection.attached) {
+          widget.focusNode.unfocus();
```

Comment:
This call seems reasonable. @HansMuller @gspencergoog any concerns for calling this when the platform disconnects the focus from the Flutter view?

Reply:
This seems fine to me.